content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -670,6 +671,8 @@ Each feature gate is designed for enabling/disabling a specific feature:
   supports project quotas and they are enabled, use project quotas to monitor
   [emptyDir volume](/docs/concepts/storage/volumes/#emptydir) storage consumption rather than
   filesystem walk for better performance and accuracy.
+- `LogarithmicScaleDown`: Enable semi-random selection of pods to evict on controller scaledown
+  based on logarithmic bucketing of pod timestamps.
```

Comment:
(just a nit)
```suggestion
- `LogarithmicScaleDown`: Enable semi-random selection of pods to evict on scale-in,
  based on logarithmic bucketing of pod timestamps.
```

Reply:
I don't think I have heard the term scale-in before. Is there precedence?

Also, this feature is specific to the replicaset controller.

Reply:
> scale-in

See https://blog.turbonomic.com/blog/on-technology/cloud-scalability-scale-vs-scale for an example of existing us.

> this feature is specific to the replicaset controller.

We should mention it applies to ReplicaSet resources.

Reply:
I'm not convinced such terminology is broadly accepted. We use up/down in other locations of our documentation https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#scaling-a-deployment

Unless you plan to change all instances.

Reply:
The idea is that _vertical_ scaling is up/down. https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#scaling-a-deployment is about horizontal autoscaling of course.

Given that the feature gate is called LogarithmicScaleDown I think it's OK to accept the mismatch. It's much too late to revise the name of the feature gate, and anyway the link between vertical and up/down might not be as widely recognized as I'd thought.

Reply:
From our events:

```
    Normal  ScalingReplicaSet  22s   deployment-controller  Scaled up replica set nginx-deployment-1564180365 to 2
    Normal  ScalingReplicaSet  19s   deployment-controller  Scaled down replica set nginx-deployment-2035384211 to 1
```

We are already using up/down for horizontal as well.